packages/react/src/components/Menu/Menu.tsx

```diff
@@ -203,6 +207,7 @@ class Menu extends AutoControlledComponent<ReactProps<MenuProps>, MenuState> {
           active,
           inSubmenu: submenu,
           indicator,
+          accessibility: (accessibility as any).childrenBehavior,
```

Comment:
what if `accessibility` is undefined?

Reply:
btw, since you're making changes in `Menu`, makes sense to add `childBehavior` for `menuBehavior`